hawkmoth/__main__.py

```diff
@@ -31,8 +31,11 @@ def main():
                         help='Verbose output.')
     args = parser.parse_args()
 
-    comments = parse_to_string(args.file, args.verbose,
-                               compat=args.compat, clang=args.clang)
+    comments, parser_errs = parse_to_string(args.file, args.verbose,
+                                            compat=args.compat, clang=args.clang)
+    if parser_errs:
+        for err in parser_errs:
+            print(err)
```

Comment:
Sphinx prints everything on stdout unfortunately, but I don't think we need to follow that here. Maybe stderr would be better here? Don't think it matters much though.

Reply:
Will use stderr then.